_source/atoms/label/Label.scss

```diff
@@ -3,4 +3,9 @@
   margin-top: 1rem;
   padding-bottom: 0.5rem;
   line-height: 1.556rem;
+  color: $color__font--dark;
+}
+
+.label--dark {
+  color: #fff;
```

Comment:
Somehow it feels confusing that `.label--dark` has the font color white 😆 Is there a possibility that other devs will work on this? Maybe we should document it somehow?

Reply:
In other cases you use `--dark-mode` as modifier, might be also the right choice here?